common/libs/keystore.js

```diff
@@ -0,0 +1,98 @@
+// @flow
+import { randomBytes, createCipheriv } from 'crypto';
+import { privateToPublic, publicToAddress, sha3 } from 'ethereumjs-util';
+import scrypt from 'scryptsy';
+import uuid from 'uuid';
+import { makeBlob } from 'libs/globalFuncs';
+
+export const scryptSettings = {
+  n: 1024
+};
+
+export const kdf = 'scrypt';
+
+export function genNewPkey(): Buffer {
+  return randomBytes(32);
```

Comment:
This stackexchange helped me understand your decision:
https://ethereum.stackexchange.com/a/3619

👍